tests/conftest.py

```diff
@@ -1,38 +1,57 @@
+import os
+import random
+import string
+
 import boxsdk
 import pytest
-import os
 
 
 ROOT_FOLDER = boxsdk.folder.Folder(
-    None, "0", {"type": "folder", "id": "0", "name": "All Files", "path_collection": {"total_count": 0, "entries": []}}
-)
-
-SHARED_FOLDER = boxsdk.folder.Folder(
     None,
-    "1",
+    "0",
     {
         "type": "folder",
-        "id": "1",
-        "name": "TestSharedFolder",
-        "path_collection": {
-            "total_count": 1,
-            "entries": [{"id": ROOT_FOLDER.id, "name": ROOT_FOLDER.name, "type": ROOT_FOLDER.type}],
-        },
+        "id": "0",
+        "sequence_id": None,
+        "etag": None,
+        "name": "All Files",
+        "path_collection": {"total_count": 0, "entries": []},
     },
 )
 
 SECRET_ARN = "arn:aws:secretsmanager:test-region:000000000000:secret:test-secret-Az1Qw2"
-SHARED_BOX_FOLDER_ID = SHARED_FOLDER.id
+SHARED_BOX_FOLDER_ID = "5"
 MANIFEST_TABLE_NAME = "test-manifest-table"
 
 
 os.environ["SECRET_ARN"] = SECRET_ARN
 os.environ["BOX_FOLDER_ID"] = SHARED_BOX_FOLDER_ID
 os.environ["MANIFEST_TABLE_NAME"] = MANIFEST_TABLE_NAME
+os.environ["AWS_DEFAULT_REGION"] = "gl-north-14"
```

Comment:
o.O What is this?

Reply:
A fake region, I needed it configured so I could get a real DynamoDB Table object, but I didn't want it to actually contact anything.  GL is the country code for Greenland.